flask_session/sessions.py

```diff
@@ -561,3 +568,13 @@ def save_session(self, app, session, response):
         response.set_cookie(app.session_cookie_name, session_id,
                             expires=expires, httponly=httponly,
                             domain=domain, path=path, secure=secure)
+
+    def delete_expired_sessions(self):
+        expired_sessions = self.sql_session_model.query.filter(
```

Comment:
I think you can simplify that to something like:

    self.sql_session_model.query.filter(self.sql_session_model.expiry < datetime.utcnow()).delete()
    self.db.session.commit()